crates/protocol/src/operator/mod.rs

```diff
@@ -4,7 +4,7 @@ use prost::Message;
 use thiserror::Error;
 use warg_crypto::{hash::DynHash, Decode, Encode, Signable};
 
-mod model;
+pub mod model;
```

Comment:
Given the other changes to this module aren't necessary, I think we can back this change out.